pkg/auth/docker/client.go

```diff
@@ -22,22 +22,26 @@ type Client struct {
 // All changes will only be written to the first config file.
 func NewClient(configPaths ...string) (auth.Client, error) {
 	var configs []*configfile.ConfigFile
```

Comment:
Once last thing is that this var can be defined after the early return (`configs`)

Reply:
Thank you 🙇